cmake/std/atdm/ats2/environment.sh

```diff
@@ -0,0 +1,231 @@
+################################################################################
+#
+# Set up env on ats2 for ATMD builds of Trilinos
+#
+# This source script gets the settings from the ATDM_CONFIG_BUILD_NAME var.
+#
+################################################################################
+
+# ats2 jobs all use the same environmnet changes to the
+# sourced script below will impact jobs on both of those
+# machines. please be mindful of this when making changes
+
+# Handle KOKKOS_ARCH
+
+if [[ "$ATDM_CONFIG_COMPILER" == "GNU"* || \
+  "$ATDM_CONFIG_COMPILER" == "XL"* ]]; then
+  if [[ "$ATDM_CONFIG_KOKKOS_ARCH" == "DEFAULT" || \
+    "$ATDM_CONFIG_KOKKOS_ARCH" == "Power9" ]] ; then
+    export ATDM_CONFIG_KOKKOS_ARCH=Power9
+    arch=pwr9
+  else
+    echo
+    echo "***"
+    echo "*** ERROR: KOKKOS_ARCH=$ATDM_CONFIG_KOKKOS_ARCH is not a valid option"
+    echo "*** for the compiler GNU.  Replace '$ATDM_CONFIG_KOKKOS_ARCH' in the"
+    echo "*** job name with 'Power9'"
+    echo "***"
+    return
+  fi
+elif [[ "$ATDM_CONFIG_COMPILER" == "CUDA"* ]] ; then
+  if [[ "$ATDM_CONFIG_KOKKOS_ARCH" == "DEFAULT" || \
+    "$ATDM_CONFIG_KOKKOS_ARCH" == "Power9" || \
+    "$ATDM_CONFIG_KOKKOS_ARCH" == "Volta70" ]] ; then
+    export ATDM_CONFIG_KOKKOS_ARCH=Volta70
```

Comment:
FYI: On 'waterman', this is set to `KOKKOS_ARCH=Power9,Volta70`.  It may not matter for SPARC, but it might matter for Trilinos.  I will test this out on my side.